### PR TITLE
Get articles comment count

### PR DESCRIPTION
**Pull request by danewhitfield**

### CODE REVIEW

File: __tests__/articles.test.js [hunk 1]
```diff
@@ -40,3 +37,40 @@ describe("GET /api/articles/:article_id", () => {
...
+        .get("/api/articles/57")
+        .expect(404)
+        .then((res) => {
+          expect(res.body).toMatchObject({ msg: "not found!" });
+        });
+    });
+  });
+});
+
+// There should be no errors if passed a valid ID with the wrong content, the comment count should just appear as 0 which is accounted for
```
Comment (P-Copley) [inline __tests__/articles.test.js:138]: Lovely comment. 😍

**Comment by P-Copley** [comment]
Lovely setup. This looks good to go 👍 

Really like the separate test files for the different endpoints.

File: app.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ app.use((err, req, res, next) => {
   if (err.msg && err.status) {
     res.status(err.status).send({ msg: err.msg });
   }
+  console.log(err);
```
Comment (P-Copley) [inline app.js:38]: You probably don't need this. Looks like it's snuck it's way in 😁

Comment (danewhitfield) [reply app.js:38]: That console log played a very vital role in this PR being possible :laughing: I will remove it after merging :D

**Pull request merged**